packages/webcert/src/feature/certificate/UnifiedEdit/UeInteger/UeInteger.tsx

```diff
@@ -26,7 +26,7 @@ const UeInteger: React.FC<Props> = ({ question, disabled }) => {
   const dispatch = useDispatch()
   const questionValue = question.value as ValueInteger
   const questionConfig = question.config as ConfigUeInteger
-  const [number, setNumber] = useState(questionValue.value?.toString() ?? '')
+  const [number, setNumber] = useState(questionValue.value?.toString() ?? undefined)
```

Comment:
Maybe a better idea would be to have the local state numeric rather than string?